### PR TITLE
fix: call .eval() on tokenizer and model in KronosPredictor

### PR DESCRIPTION
**Pull request by hexonal**
KronosPredictor never switches the modules out of training mode, so inference runs with dropout active.

## Impact

Kronos-small ships `attn_dropout_p = 0.1`. Because `KronosPredictor.__init__` only does `.to(device)`, the modules stay in `train()` mode for the whole lifetime of the predictor:

- **On MPS it crashes outright.** `F.scaled_dot_product_attention` with a non-zero `dropout_p` is not implemented for the MPS backend, so any `predict()` on Apple Silicon raises rather than returning a forecast.
- **On CUDA/CPU it is worse than a crash — it is silent.** Predictions come back looking fine, but every forward pass drops 10% of the attention weights at random. Two calls with identical inputs give different answers for a reason that has nothing to do with the models sampling behaviour, and the extra variance is indistinguishable from the intended `sample_count` sampling noise.

Measured on this fork: with `sample_count=5` on 60m US equity bars, adding `.eval()` cut MAE by roughly a fifth on an otherwise unchanged pipeline.

## Fix

```python
self.tokenizer = self.tokenizer.to(self.device).eval()
self.model = self.model.to(self.device).eval()
```

`.eval()` is the conventional place for this — the predictor is inference-only, and nothing in the class ever calls `.train()`. Callers who genuinely want stochastic modules can still flip them back after construction.

Two lines, no behavioural change for anyone who was already calling `.eval()` themselves.

### CODE REVIEW

**Review by itsadrianxv — COMMENTED** [review]
Good catch — this fixes the footgun where dropout stays active during inference unless the caller remembers to call `.eval()` (related to the cone-width discussion in #387/#391). Since `KronosPredictor` is inference-only, defaulting to eval mode in the constructor is the right place.

One optional note: anyone reusing this class for fine-tuning would need to switch back to `.train()` explicitly — maybe worth a one-line docstring mention, but not blocking. LGTM.
